### ISSUE TITLE
"Hello, World!" exercises of tracks, requesting mentorship link

### ISSUE THREAD

**Issue by myucesan** [exercism/exercism#6026]
Hello there,

The Hello World exercises of tracks show a request mentoring link that doesn't work for obvious reasons. 
<img width="1205" alt="image" src="https://user-images.githubusercontent.com/4138641/136673192-bcdb49e4-1e0a-42ba-84e5-6ccc845a8ce2.png">

When clicked it reloads the page, but should be totally gone.

Yours sincerely,

myucesan

**Comment by kotp** [comment]
I do not think it should be totally gone.  It should work, because there are students being introduced to the workflow of Exercism and this is a great place for them to request help  if they can not get it to work!

If I see a Hello World exercise that is requesting help, then I know it is likely that we are not clear enough in the instructions delivered, or they need further help.

**Comment by kotp** [comment]
![image](https://user-images.githubusercontent.com/16747/136673315-c9595a12-ecf8-44f0-ba9b-d7af54361a89.png)

The "get mentoring" button below is not activated either, and this may be related in terms of usability.
